### ISSUE TITLE
Serverless plugin "serverless-plugin-localstack" not found. 

### ISSUE THREAD

**Issue by panagath** [temyers/serverless-localstack#14]
I get the following error.

I have added the  ```.serverless_plugins``` directory in my project at the same lever where I have the serverless.yml file . Inside this folder I have tried various combinations, as in the docs, pointing to the main directory where I checked out the localstack plugin, by pointing directly to the index.js file, by copying the index.js and renaming it to serverless-plugin-localstack etc...always the same error. 

Ofcourse I have the configuration in the serverless.yml as in the docs

How do we configure it exactly? the documentation is very vague

```
Serverless Error ---------------------------------------

  Serverless plugin "serverless-plugin-localstack" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.
```

**Comment by dinhtuan1991vt** [comment]
i have this issue, too

**Comment by panagath** [comment]
Try this:

Download it manually

Run in terminal: 
`npm install bluebird --user`
`npm install aws-sdk --user`

cd in the plugin directory and inside the src folder run

`npm build`

Then run: `npm list`.    And check that serverless-localstack is in there

Inside your project’s root directory do `mkdir .serverless_plugins` .  And then cd in that directory and  run
`ln -s /<PATH TO THE PLUGIN DIRECTORY>/src/index.js serverless-plugin-localstack`

make sure localstack is running and the deploy serverless

**Comment by justinm** [comment]
Try using serverless-localstack instead of serverless-plugin-localstack

**Comment by gulbinas** [comment]
i have same issue too, but found this
https://github.com/localstack/localstack

**Comment by temyers** [comment]
Now published to NPM:

https://www.npmjs.com/package/serverless-localstack

```
npm install -g serverless-localstack
```

**Comment by abuzarhamza** [comment]
I am encountering the same issue . Installed the serverlss-localstack globally, yet giving the error

**Comment by temyers** [comment]
The plugin definition in the `serverless.yml` should be `serverless-localstack`, not `serverless-plugin-localstack`

The README needs updating

**Comment by flaviocordova** [comment]
I've followed the steps (to install using npm, not building locally) and I still get the same error...

```
  Serverless Error ---------------------------------------

  Serverless plugin "serverless-localstack" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless

  Your Environment Information -----------------------------
     OS:                     darwin
     Node Version:           8.7.0
     Serverless Version:     1.25.0
```

Am I missing something ?

**Comment by temyers** [comment]
Can you include the following:
- your package.json (dependencies)
- does `npm install --save-dev serverless-localstack` complete successfully?

**Comment by rafakato** [comment]
Hello, I think this might be the problem: https://github.com/temyers/serverless-localstack/blob/master/.npmignore#L7

Adding the `src` to npm ignore makes npm not copy the `src` folder when installing, that is what I see when navigating to the `node_modules` folder for `serverless-localstack`.

**Comment by flaviocordova** [comment]
Hi, everybody...

@temyers, I can install it without any problem:

```
npm install --save-dev serverless-localstack
+ serverless-localstack@0.2.1

```

and here's my package.json

```
{
  "name": "test-app",
  "version": "0.0.1",
  "description": "Test App",
  "author": "flavio.cordova",
  "private": true,
  "main": "index.js",
  "scripts": {
    "dev": "node serve-local.js",
    "dev-api": "node serve-local.js api",
    "build": "nuxt build",
    "localstack": "nuxt build && serverless deploy --stage localstack",
    "deploy-dev": "nuxt build && serverless deploy --stage dev",
    "deploy-dev-fast": "nuxt build && serverless deploy --stage dev --function main",
    "deploy-prod": "nuxt build && serverless deploy --stage prod"
  },
  "dependencies": {
    "@nuxtjs/axios": "^4.5.2",
    "aws-sdk": "^2.179.0",
    "aws-serverless-express": "^3.0.0",
    "axios": "^0.16.2",
    "dynamodb": "^1.0.0",
    "dynamoose": "^0.8.6",
    "express": "^4.15.2",
    "express-session": "^1.15.6",
    "nuxt": "^1.0.0",
    "passport": "^0.4.0",
    "passport-linkedin-oauth2": "^1.5.0",
    "passport-oauth": "^1.0.0",
    "uuid": "^3.1.0",
    "vue": "^2.5.9",
    "vue-authenticate": "^1.3.4",
    "vue-axios": "^2.0.2",
    "vuetify": "=1.0.0-alpha.3",
    "vuex-persistedstate": "^2.4.2"
  },
  "devDependencies": {
    "babel-eslint": "^7.2.3",
    "eslint": "^4.3.0",
    "eslint-config-standard": "^10.2.1",
    "eslint-loader": "^1.9.0",
    "eslint-plugin-html": "^3.1.1",
    "eslint-plugin-import": "^2.7.0",
    "eslint-plugin-node": "^5.1.1",
    "eslint-plugin-promise": "^3.5.0",
    "eslint-plugin-standard": "^3.0.1",
    "serverless-localstack": "^0.2.1",
    "stylus": "^0.54.5",
    "stylus-loader": "^3.0.1",
    "vue-server-renderer": "^2.5.9",
    "vue-template-compiler": "^2.5.9",
    "yamljs": "^0.3.0"
  }
}
```

@rafakato, thanks for your reply.. I don't have a .npmignore file in this project.

**Comment by RyanMcDonald** [comment]
@rafakato is correct. The package.json for `serverless-localstack` has `"main": "src/index.js",`. But when you install `serverless-localstack` with npm, index.js does not get installed. It basically installs an empty package.

**Comment by temyers** [comment]
Fixed in 0.2.2
